src/com/google/enterprise/adaptor/database/ValidatedUri.java

```diff
@@ -54,10 +54,6 @@ public ValidatedUri(String uriString) throws URISyntaxException {
       throw new URISyntaxException(uriString, reason);
     }
 
-    if (!uri.isAbsolute()) {
```

Comment:
seeing how we missed that URL ctor detects this condition should we annotate ctor with someting like "// prevents relative URL"